app/lib/three_scale/semantic_form_builder.rb

```diff
@@ -4,6 +4,8 @@ class SemanticFormBuilder < ::Formtastic::FormBuilder
 
     delegate :tag, :site_account, :controller, to: :template
 
+    alias bot_protection bot_protection_inputs
```

Comment:
What would happen if we just change from `bot_protection` to `bot_protection_inputs` here?
https://github.com/3scale/porta/blob/master/app/views/provider/signups/_form.html.slim#L16
https://github.com/3scale/porta/blob/9debd91d45153066a9d01c9baf070b15801d05f0/test/integration/lib/three_scale/bot_protection/form_test.rb#L25

Also looks like this method wouldn't be necessary in this case:
https://github.com/3scale/porta/blob/9debd91d45153066a9d01c9baf070b15801d05f0/test/integration/lib/three_scale/bot_protection/form_test.rb#L16-L18

And neither this alias.

That's my hope :sweat_smile:

Reply:
I fixed it in: https://github.com/3scale/porta/pull/3681/commits/cb600bfba98d2d8bd88d75ab3ed8932f36b2bac7

Reply:
Now let's just pray that the tests pass :grimacing:

Reply:
Of course they passed 😎